docs/configuration/network_filters/mongo_proxy_filter.rst

```diff
@@ -151,6 +178,12 @@ mongo.logging_enabled
   % of messages that will be logged. Defaults to 100. If less than 100, queries may be logged
   without replies, etc.
 
+mongo.fault.fixed_delay.percent
+  % of requests that will be delayed. Defaults to the *percent* specified in the config.
```

Comment:
% of requests is incorrect. This would be the number of 'first' incoming data that is received and the delayenabled function(name not sure is correct) says a delay should be injected.